Backend/Underdogs-Struggling/src/main/java/com/TheTroisMousquetaires/Underdogs/Struggling/payload/security/LoginDto.java

```diff
@@ -0,0 +1,30 @@
+package com.TheTroisMousquetaires.Underdogs.Struggling.payload.security;
+
+public class LoginDto {
+    private String email;
+    private String password;
+
+    public LoginDto() {
+    }
+
+    public LoginDto(String accountOrEmail, String password) {
+        this.email = accountOrEmail;
+        this.password = password;
```

Comment:
Had a quick discussion on why "accountOrEmail" is used instead of just "email." Lots of apps allow users to sign up using email/phone number but will still generate a UserID that may or may not be shared with user. Still an opened question for now on how to approach/implement this later so will leave it for now with a note.

Reply:
Create a new issue for this problem: https://github.com/RunpengNie/underdog-s-struggling/issues/35